src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```diff
@@ -419,6 +458,26 @@ describe('ProgrammingExerciseUpdateComponent', () => {
             route.url = of([{ path: 'import' } as UrlSegment]);
         });
 
+        it('should correctly import into an exam exercise', fakeAsync(() => {
+            const examId = 1;
+            const exerciseGroupId = 1;
+            const exerciseGroup = new ExerciseGroup();
+            exerciseGroup.id = exerciseGroupId;
+            exerciseGroup.exam = { id: examId, course };
+            const programmingExercise = new ProgrammingExercise(undefined, exerciseGroup);
+            jest.spyOn(exerciseGroupService, 'find').mockReturnValue(of(new HttpResponse({ body: exerciseGroup })));
+            route = TestBed.inject(ActivatedRoute);
+            route.params = of({ courseId, examId, exerciseGroupId });
+            route.data = of({ programmingExercise });
+
+            comp.ngOnInit();
+            tick();
+            expect(comp.programmingExercise.exerciseGroup).toBe(exerciseGroup);
+            expect(comp.programmingExercise.course).toBeUndefined();
+            expect(comp.isImportFromExistingExercise).toBeTruthy();
+            expect(comp.isExamMode).toBeTruthy();
```

Comment:
```suggestion
            expect(comp.isImportFromExistingExercise).toBeTrue();
            expect(comp.isExamMode).toBeTrue();
```
Does `undefined` need to be handled?